talos.nf

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl=2
+
+include { VcfToMt } from './modules/talos/VcfToMt/main'
+include { ConvertPedToPhenopackets } from './modules/talos/ConvertPedToPhenopackets/main'
+include { MakePhenopackets } from './modules/talos/MakePhenopackets/main'
+include { GeneratePanelData } from './modules/talos/GeneratePanelData/main'
+include { QueryPanelapp } from './modules/talos/QueryPanelapp/main'
+include { FindGeneSymbolMap } from './modules/talos/FindGeneSymbolMap/main'
+include { RunHailFiltering } from './modules/talos/RunHailFiltering/main'
+include { ValidateMOI } from './modules/talos/ValidateMOI/main'
+include { HPOFlagging } from './modules/talos/HPOFlagging/main'
+include { CreateTalosHTML } from './modules/talos/CreateTalosHTML/main'
+
+
+process RunHailFilteringSV {
+    // runs the hail structural-variant filtering
+    // here we can have multiple files - should we derive output name from input?
+    publishDir 'results', mode: 'copy'
+
+    input:
+        path matrix_table
+        path panelapp_data
+        path pedigree
+
+    output:
+        path "${params.cohort}_small_variants.vcf.bgz", emit: "vcf"
+        path "${params.cohort}_small_variants.vcf.bgz.tbi", emit: "index"
```

Comment:
Unless you have a good reason to want to handle these separately, you could emit them as a `tuple` so that the index always remains paired with the VCF

Reply:
Oh yeah, I meant to circle back to this

Reply:
ah, I didn't update this because I'm not using it anyway. Deleted per the other comment